src/Selenium/Combinators.purs

```diff
@@ -33,7 +32,7 @@ tryFind probablyLocator =
 waitUntilJust :: forall e o a. Selenium e o (Maybe a) -> Int -> Selenium e o a
 waitUntilJust check time = do
   wait (checker $ isJust <$> check) time
-  fromJust <$> check
+  check >>= maybe (throwError $ error $ "Maybe was not Just after waiting for isJust") pure
```

Comment:
I don't particularly like this `throwError` here, but it seemed preferable to making the function partial. From my understanding of the code, it looks like the maybe would always be `Just` at this point in the code. However, because wait takes `Selenium e o Boolean`, we can't extract the `a` and return it as a part of checking whether it's `Just` or not. What we really want is a wait function that takes a `Selenium e o (Maybe a)` and pattern matches the `Maybe` to extract the `a` and only "returns" when the Just happens. That way you can write it non-partially and return a `Selenium e o a`.

I followed the breadcrumbs and it looks like the underlying webdriver wait function needs a `-> bool` function, so we might be out of luck, but you guys might know of a way.